docs/19_security_audit_document.md

```diff
@@ -0,0 +1,230 @@
+|`duration`    | A duration of this build excluding the queue time.          |
+|`projectId`   | An id of the project this build belongs to.                 |
+|`startedAt`   | A date-time of this build's start.                          |
+|`url`         | A URL of the source control revision used to run the build. |
+|`workflowName`| A name of the workflow executed this build.                 |
+
+Here is a table of security rules applied to the `build` collection.
+
+| Operation          | Security Rules                  |
+|--------------------|---------------------------------|
+| `read`             | `Authorization`                 |
+| `create`, `update` | `Authorization`, `isBuildValid` |
+| `delete`           | `Prohibited`                    |
+
+### The `feature_config` collection
+
+The `feature_config` collection contains the configuration values to enable or disable the application features. This collection contains a single `feature_config` document containing all configurations for the whole application.
+
+Consider the following table that describes the fields of the `feature_config` document:
+
+| Field                           | Description                                                           |
+|---------------------------------|-----------------------------------------------------------------------|
+| `isDebugMenuEnabled`            | Indicates whether the `Debug Menu` feature is enabled.                |
+| `isPasswordSignInOptionEnabled` | Indicates whether the `Email and Password` sign in option is enabled. |
+
+Here is a table of security rules applied to the `feature_config` collection.
+
+| Operation | Security Rules |
+|-----------|----------------|
+| `read`    | `Allowed`      |
+| `write`   | `Prohibited`   |
+
+### The `project_groups` collection
+
+The `project_groups` collection contains project group data. The single document of this collection contains data of a specific project group.
+
+Consider the following table that describes the fields of the `project_groups` document:
+
+| Field        | Description                                               |
+|--------------|-----------------------------------------------------------|
+| `name`       | A name of this project group.                             |
+| `projectIds` | A list of project IDs that belong to this project group.  |
+
+Here is a table of security rules applied to the `project_groups` collection.
+
+| Operation         | Security Rules                         |
+|-------------------|----------------------------------------|
+| `read`, `delete`  | `Authorization`                        | 
+| `create`, `update`| `Authorization`, `isProjectGroupValid` | 
+
+### The `projects` collection
+
+The `projects` collection defines projects within the Metrics Web Application. The single document stands for one project and contains the project's name.
+
+Here is a table of security rules applied to the `projects` collection.
+
```

Comment:
Does it make sense to specify the fields of this document stored in the database to be more consistent? The same for the `user_profiles` collection.